src/components/sections/judge/JudgeContestsTableRow.tsx

```diff
@@ -73,7 +73,9 @@ const JudgeContestsTableRow = ({
     ) {
       return (
         <Button asChild variant="outline" size="small">
-          <Link href="#" className="gap-2 text-buttonS">
+          <Link
+            href={PATHS.judgeContestFindings(contest.id)}
```

Comment:
The issue is, that this is the only place where I can access this page. If a judge approves all the findings, it's no longer possible to view them. Maybe consider navigating to this page when clicking the contest title, or adding another button.

Reply:
True. I added a link to the pending, approved and rejected findings count in the table rows 👍

Reply:
Actually it's quite difficult to spot. I think a better way would be to always have "Review rewards" button here, and move the "Finalize rewards" button to the judge contest finding page. That way it's also more convenient, when you finish reviewing the findings, you can immediately click on "Finalize" without having to move back.